change README to tell about coverage only, not full project config (#2)
sonarsource-cfamily-examples/windows-msbuild-vscoverage-azure-sc: README.adoc

> = C++ example project for generating test coverage with vscoverage on Azure Pipelines

Reviewer: ```suggestion
= C++ example project for generating test coverage with VisualStudio coverage tool on Azure Pipelines
```
IMHO vscoverage isn't a thing, and we can afford to spell the full name of the concept

image:{img-build-status}[Build Status, link={uri-build-status}]
image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]

*The test coverage shown here works on both SonarQube and SonarCloud. This project is analysed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_windows-vscoverage-msbuild-azure-sc[SonarCloud]!*

NOTE: This simple example aims at demonstrating code coverage in action. It explains how to add coverage on a project for which the analysis is already configured. Some analysis https://github.com/orgs/sonarsource-cfamily-examples/repositories?q=topic%3Aconfiguration+msbuild-azure&type=all&language=&sort=[configuration examples] are available. +
Adding code coverage is done the same way regardless of whether you use the Build Wrapper or a Compilation Database to configure the analysis.

> To get coverage and feed it to the sonar-scanner, use the following procedure.

Reviewer: ```suggestion
To get coverage and feed it to the Sonar Scanner, use the following procedure.
```

> . In your azure-pipelines.yml file, Initialize the extra property `sonar.cfamily.vscoveragexml.reportsPath` with the path that will contain your XML coverage report

Reviewer: ```suggestion
. In your azure-pipelines.yml file, initialize the extra property `sonar.cfamily.vscoveragexml.reportsPath` with the path that will contain your XML coverage report
```

. Add a `VSTest` task to launch your tests. Be sure to enable code coverage and to convert the result file into an XML file

> . Add the `SonarCloudAnalyze` task

Reviewer: ```suggestion
. Make sure the `SonarCloudAnalyze` task runs after the `VSTest` task
```
If we assume the analysis is configured, `SonarCloudAnalyze` task is already there.

- https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-steps-using-compilation-database[Using compilation database (compile_commands.json)]
- https://docs.sonarcloud.io/enriching/test-coverage/test-coverage-parameters/[Test coverage parameters]

> = Windows\MSBuild

Reviewer: Why remove this section?

Author: as agreed, closing this

Reviewer: After f2f discussion. It is no longer relevant.
